src/trexio_module.F

```diff
@@ -0,0 +1,17 @@
+!--------------------------------------------------------------------------------------------------!
```

Comment:
I don't see any USE statements for this module file. Is it really needed?

Reply:
I talked with the developer of TREXIO and we figured that this is the (best?) way to use their fortran API. The library is written in C and compiled by the toolchain, and the fortran API is simply the module source code available in `fortran_f.f90`, which you have to compile yourself together with your program. `trexio_f.f90` is located in the `tools/toolchain/install/trexio` directory.
I don't know whether there is a better way to do this..?

Reply:
Ok, that case I'd suggest you rename the file from `trexio_module.F` to `trexio.F`. This way it becomes clear that the USE-statement in `trexio_utils.F` refers to this file.